src/lib/profiles/data-management/Current/NotificationEngine.cpp

```diff
@@ -1850,3 +1855,23 @@ void NotificationEngine::Run()
 
     return;
 }
+
+WEAVE_ERROR NotificationEngine::EnableNotifications(void)
+{
+    WEAVE_ERROR err  = WEAVE_NO_ERROR;
+
+    WeaveLogDetail(DataManagement, "<NE> Enabling notifications!");
+    mNotifyTxEnabled = true;
+
+    return err;
+}
+
+WEAVE_ERROR NotificationEngine::DisableNotifications(void)
+{
+    WEAVE_ERROR err  = WEAVE_NO_ERROR;
+
+    WeaveLogDetail(DataManagement, "<NE> Disabling notifications!");
```

Comment:
Progress level?

Reply:
For this one, I am keeping as Detail to avoid more logging increase and also, because it, trivially, is setting a member.